### PR TITLE
WIP: TMA-1192: dependable CI env

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: lcm.rake [hunk 1]
```diff
@@ -74,35 +74,28 @@ end
...
-  end
-
-  namespace :integration do
-    desc 'Run integration tests in Docker'
-    task :docker do
-      system('docker-compose -f docker-compose.lcm.yml up --force-recreate --abort-on-container-exit --exit-code-from appstore appstore') ||
+  namespace :docker do
+    desc 'Run integration and slow tests in Docker'
+    task :integration do
+      system('docker-compose -f docker-compose.lcm.yml up --force-recreate --abort-on-container-exit --exit-code-from appstore appstore bundle exec rake -f lcm.rake test:integration') ||
```
Comment (yenkins) [inline lcm.rake]: warning: Metrics/LineLength: Line is too long. [186/180] (rubocop)

File: lcm.rake [hunk 2]
```diff
@@ -74,35 +74,28 @@ end
...
-    desc 'Run load tests in Docker'
-    task :docker do
-      system('docker-compose -f docker-compose.lcm.yml run --rm appstore bundle exec rake -f lcm.rake test:load')
 
-      system(check_exit_code) || fail('Test execution failed!')
-    end
+  desc 'Run load tests in Docker'
+  task :load do
+    system('docker-compose -f docker-compose.lcm.yml run --rm appstore bundle exec rake -f lcm.rake test:load') ||
+        fail('Test execution failed!')
```
Comment (yenkins) [inline lcm.rake]: warning: Layout/MultilineOperationIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines. (rubocop)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/883/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1696/ : FAILURE in 1m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1957/ : SUCCESS in 1m 12s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/134/ : SUCCESS in 8m 26s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/884/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1697/ : FAILURE in 1m 13s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1958/ : SUCCESS in 1m 25s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/135/ : SUCCESS in 7m 38s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/885/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1698/ : FAILURE in 1m 10s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1959/ : SUCCESS in 1m 14s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/136/ : SUCCESS in 7m 31s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/889/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1702/ : FAILURE in 2m 57s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1963/ : SUCCESS in 2m 44s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/140/ : SUCCESS in 7m 12s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/891/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1704/ : FAILURE in 1m 18s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1965/ : SUCCESS in 1m 20s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/142/ : SUCCESS in 11m 06s

**Pull request closed without merging**